scripts/dump-metrics-csv.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import codecs
+import fnmatch
+import os
+import sys
+import yaml
+
+all_metrics = ""
+
+for root, dirs, files in os.walk("."):
+    for file in files:
+        if fnmatch.fnmatch(file, "metadata.yaml"):
+             with open(os.path.join(root, file)) as f:
+                data = yaml.load(f)
```

Comment:
Use `yaml.load_safe` so we don't get complains about unsafe things in security scanners.